docs/prod-install.md

```diff
@@ -70,34 +67,24 @@ symbolic link to it.
 * Configures the various third-party services Zulip uses, including
 Postgres, RabbitMQ, Memcached and Redis.
 
-## Step 3: Configure Zulip
+#### Providing your own SSL certificate
 
-Configure the Zulip server instance by editing `/etc/zulip/settings.py` and
-providing values for the mandatory settings, which are all found under the
-heading `### MANDATORY SETTINGS`.  These settings include:
+If you'd like to use an SSL certificate that you obtained not using
+Certbot way (e.g. issued by your corporate certificate authority),
+[our ssl certificate documentation](ssl-certificates.html) covers what
+you need to do.
 
-- `EXTERNAL_HOST`: the user-accessible Zulip domain name for your
-  Zulip installation (i.e., what users will type in their web
-  browser). This should of course match the DNS name you configured to
-  point to your server and for which you configured SSL certificates.
-  If you plan to use multiple domains, add the others to
-  `ALLOWED_HOSTS`.
+## Step 3: Configure outgoing email
 
-- `ZULIP_ADMINISTRATOR`: the email address of the person or team
-  maintaining this installation and who will get support and error
-  emails.
+Configure the Zulip server instance by editing
+`/etc/zulip/settings.py` to enable your server's ability to send
+outgoing emails:
 
 - `EMAIL_HOST`, `EMAIL_HOST_USER`: credentials for an outgoing email
```

Comment:
Was about to raise that there is dictionary that maps domain name to its smtp.domain name version: e.g. for gmail https://github.com/nylas/sync-engine/blob/b91b94b9a0033be4199006eb234d270779a04443/inbox/providers.py#L311. Perhaps this could simplify the config so that a user just needs to enter the sender email and the sending server is then auto-inferred.

Reply:
Hmm. I'm not sure this helps that much -- most of the data there isn't relevant for Zulip, and sending from something like a `gmail.com` address doesn't seem super likely anyway.